easybuild/easyconfigs/p/Philosopher/Philosopher-5.0.0-GCC-11.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'Philosopher'
+version = '5.0.0'
+
+homepage = 'https://philosopher.nesvilab.org/'
+description = """Philosopher is a fast, easy-to-use, scalable, and versatile
+ data analysis software for mass spectrometry-based proteomics. Philosopher
+ is dependency-free and can analyze both traditional database searches and
+ open searches for post-translational modification (PTM) discovery."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/Nesvilab/philosopher/archive/']
+sources = [{'download_filename': 'v%s.tar.gz' % version, 'filename': SOURCE_TAR_GZ}]
+checksums = ['0c6895c60c1c2543f7b0ad7f8baae4be39c9c519d6035c8fdd2ad8a2f83a99cc']
+
+builddependencies = [
+    ('Go', '1.21.2', '', SYSTEM),
+]
+
+# see https://github.com/Nesvilab/philosopher/wiki/How-to-Build
+buildopts = 'CC="$CC" CFLAGS="$CFLAGS" deploy && go build -o philosopher'
```

Comment:
Try this instead:
```python
prebuildopts = 'export CC="$CC" CFLAGS="$CFLAGS" && '
buildopts = "deploy && make philosopher"
```

This will result in a statically linked `philosopher` binary, see also https://github.com/Nesvilab/philosopher/blob/14285764ebaf3c07eb78c24a154cb2330a541a05/Makefile#L12